scripts/functions.js

```diff
@@ -503,7 +503,9 @@ function zoomToPolygon(sourceID, coords, filterField, maskLayer = true) {
 		// reset dropdowns as appropriate
 		if (sourceID !== "state-school-districts") {
 			showSchoolDistricts = false;
-			document.getElementById("school-districts-control").tomselect.clear(true);
+			if (document.getElementById("school-districts-control").tomselect) {
```

Comment:
This is tiny, but I suggest assigning `document.getElementById("school-districts-control")` to a variable for the if statement and then using it on line 507 if needed because DOM queries are actually kind of expensive (although an ID query is the cheapest kind)

Reply:
Oh, interesting.  I don't have a good sense of the cost of operations like that, I think mainly because I'm used to map operations always being a bigger term.  But it seems like a good habit to be in so I've made the change, and found a few other `document.getElementById()` calls to de-duplicate - some of which won't affect performance at all because they're in different branches of a conditional, but that'll be better for future code maintenance.

Reply:
whoa, nice!